### PR TITLE
Rsx debug: UI improvement and more opcode disasm supported

### PR DESCRIPTION
**Pull request by vlj**
This PR adds the ability to view color content in bigger window. It also support pretty printing of texture registers.

### CODE REVIEW

**Comment by AlexAltea** [comment]
I would prefer to be filtered out from @mention-bot notifications. Thanks. :-)

**Comment by tambry** [comment]
@AlexAltea I'll be adding you to the blacklist.

**Comment by danilaml** [comment]
it also fixes Apple Clang Bug (?) that cause over 40 mins of build times on travis OSX by moving big static unordered_maps into cpp, now they back to normal ~12 mins.

**Comment by vlj** [comment]
rebased

File: rpcs3/Gui/RSXDebugger.cpp [hunk 1]
```diff
@@ -341,10 +358,15 @@ void RSXDebugger::OnClickBuffer(wxMouseEvent& event)
 		return; \
 	} \
 
-	if (event.GetId() == p_buffer_colorA->GetId()) SHOW_BUFFER(0);
+/*	if (event.GetId() == p_buffer_colorA->GetId()) SHOW_BUFFER(0);
 	if (event.GetId() == p_buffer_colorB->GetId()) SHOW_BUFFER(1);
```
Comment (Nekotekina) [inline rpcs3/Gui/RSXDebugger.cpp]: Don't use /*

File: rpcs3/Gui/RSXDebugger.cpp [hunk 2]
```diff
@@ -323,6 +323,23 @@ void RSXDebugger::OnScrollMemory(wxMouseEvent& event)
 	event.Skip();
 }
 
+namespace
+{
```
Comment (Nekotekina) [inline rpcs3/Gui/RSXDebugger.cpp]: Why namespace if marking display_buffer static will have the same effect? IIRC.

File: rpcs3/Emu/RSX/GCM.cpp [hunk 3]
```diff
@@ -0,0 +1,1269 @@
...
+		case CELL_GCM_TEXTURE_A4R4G4B4: return "CELL_GCM_TEXTURE_A4R4G4B4";
+		case CELL_GCM_TEXTURE_R5G6B5: return "CELL_GCM_TEXTURE_R5G6B5";
+		case CELL_GCM_TEXTURE_A8R8G8B8: return "CELL_GCM_TEXTURE_A8R8G8B8";
+		case CELL_GCM_TEXTURE_COMPRESSED_DXT1: return "CELL_GCM_TEXTURE_COMPRESSED_DXT1";
+		case CELL_GCM_TEXTURE_COMPRESSED_DXT23: return "CELL_GCM_TEXTURE_COMPRESSED_DXT23";
+		case CELL_GCM_TEXTURE_COMPRESSED_DXT45: return "CELL_GCM_TEXTURE_COMPRESSED_DXT45";
+		case CELL_GCM_TEXTURE_G8B8: return "CELL_GCM_TEXTURE_G8B8";
+		case CELL_GCM_TEXTURE_R6G5B5: return "CELL_GCM_TEXTURE_R6G5B5";
+		case CELL_GCM_TEXTURE_DEPTH24_D8: return "CELL_GCM_TEXTURE_DEPTH24_D8";
+		case CELL_GCM_TEXTURE_DEPTH24_D8_F
```
Comment (Nekotekina) [inline rpcs3/Emu/RSX/GCM.cpp]: No newline

**Pull request merged**